posthog/models/activity_logging/activity_log.py

```diff
@@ -78,14 +81,34 @@ class Meta:
     created_at: models.DateTimeField = models.DateTimeField(default=timezone.now)
 
 
-field_exclusions: Dict[Literal["FeatureFlag", "Person"], List[str]] = {
+field_exclusions: Dict[Literal["FeatureFlag", "Person", "Insight"], List[str]] = {
     "FeatureFlag": ["id", "created_at", "created_by", "is_simple_flag",],
     "Person": ["id", "uuid", "distinct_ids", "name", "created_at", "is_identified",],
+    "Insight": [
+        "id",
+        "filters_hash",
+        "created_at",
+        "refreshing",
+        "dive_dashboard",
+        "updated_at",
+        "type",
+        "funnel",
+        "deprecated_tags",
+        "last_modified_at",
+        "layouts",
+        "color",
+        "order",
+        "result",
+        "dashboard",
```

Comment:
Should we track `dashboards` now?

Reply:
To check on these two bits of review...

This is a list of things to exclude on the server. So it says "don't track these deprecated fields" 

Have you misunderstood the code or have I misunderstood the review 🤣 

(we do pick up dashboards field in changes)

Reply:
imagine I put a homer bush gif here